libs/utils/trace.py

```diff
@@ -629,14 +629,27 @@ def _sanitize_SchedEnergyDiff(self):
             'Suboptimal Reject')
 
     def _sanitize_SchedOverutilized(self):
-        """ Add a column with overutilized status duration. """
+        """
+        Add a column with overutilized status duration.
+        """
         if not self.hasEvents('sched_overutilized'):
             return
+
         df = self._dfg_trace_event('sched_overutilized')
         df['start'] = df.index
         df['len'] = (df.start - df.start.shift()).fillna(0).shift(-1)
         df.drop('start', axis=1, inplace=True)
 
+        # Fix the last event which will have a NaN duration
+        df.ix[-1, "len"] = self.start_time + self.time_range - df.index[-1]
+
+        # Build a stat on trace overutilization
+        self.overutilized_time = df[df.overutilized == 1].len.sum()
+        self.overutilized_prc = 100. * self.overutilized_time / self.time_range
+
+        self._log.debug('Overutilized time: %.6f [s] (%.3f%% of trace time)',
+                            self.overutilized_time, self.overutilized_prc)
```

Comment:
I think the lines from "Build a stat" onwards are supposed to be added in the previous commit?